skfda/misc/metrics/_lp_norms.py

```diff
@@ -134,7 +133,9 @@ def integrand(*args: NDArrayFloat) -> NDArrayFloat:  # noqa: WPS430
                 except Exception:
                     f1 = call(f_args)
 
-                return np.power(np.abs(f1), self.p)
+                return np.asarray(
+                    np.power(np.abs(f1), self.p), dtype=np.float64,
```

Comment:
Why do we need to fix the width of the float dtype here? We normally do not do this.